Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -158,14 +170,15 @@ upload_stream () {
   esac
 
   while true; do
-    while read -r -d "$delimiter" line; do
+    while read -r -d "$delimiter" line || [[ -n "$line" ]]; do
       n=`ps ax | grep 'curl' | grep "$HOST:$PORT" | grep -v 'grep' | wc -l`
       while [ $n -gt $CURL_N_MAX ]; do
         sleep $SLEEP
         n=`ps axf | grep 'curl' | grep "$HOST:$PORT" | grep -v 'grep' | wc -l`
       done
       echo "$line" | $cmd &>/dev/null || { echo "(ERROR) An error occured while uploading stream data." >&2; continue; } &
-      echo -n $'\6'
+      echo -ne EOMessage
```

Comment:
`EOMessage` here looks a bit excessive -- Output Connector Stage (here -- Stage 60) sends no data back to supervisor program, so there\`s no messages to be separated.